MetaMask/metamask-mobile
# feat: adjusting the 'add network screen' in network bottom sheet, for network UI redesign

## **Description**
This PR shows the 'Add custom network' form directly after clicking the button, where the user can add and connect to a custom network. A demo can be viewed here: https://www.loom.com/share/bd380aca1065428ea584b0421ce69fb9

Here is a link to the figma: https://www.figma.com/design/76R5BvAVubUhWaN2Ld7MAv/Default-Networks?node-id=1303-34400&m=dev


## **Related issues**

Ensures that the form is displayed directly after clicking since the list of popular networks are now integrated in the list.

## **Manual testing steps**

1. From the home page, click the network selector at the top of the screen, a bottom sheet comes up with a list of networks
2. At the bottom of the bottom sheet, click Add custom network button, a form should show up allowing to add a network
3. Fill the form with relevant information and add the network

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
